Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -280,6 +280,23 @@ public extension UIView {
 	public func addSubviews(_ subviews: [UIView]) {
 		subviews.forEach({ self.addSubview($0) })
 	}
+    
+    /// SwifterSwift: Add array of subviews to view and set translatesAutoresizingMaskIntoConstraints.
+    ///
+    /// - Parameter subviews: array of subviews to add to self.
+    /// - Parameter translatesAutoresizingMaskIntoConstraints: set translatesAutoresizingMaskIntoConstraints of the added subviews.
+    public func addSubviews(_ subviews: [UIView], translatesAutoresizingMaskIntoConstraints: Bool) {
+        subviews.forEach({ self.addSubview($0, translatesAutoresizingMaskIntoConstraints: translatesAutoresizingMaskIntoConstraints) })
```

Comment:
Nothing to say, it's simple as it should be. Maybe we should consider removing that "self" before addSubview ?
I'm not a fan of it when unnecessary and in the single view and in the methods it is not adopted. I feel like we should follow a common style.

Reply:
sure, done! Fixed it in the other `addSubviews` method as well